src/main/java/essenmakanan/command/AddIngredientCommand.java

```diff
@@ -5,18 +5,20 @@
 import essenmakanan.recipe.RecipeList;
 
 public class AddIngredientCommand extends Command{
+    private String toAdd;
 
-    public AddIngredientCommand() {
+    public AddIngredientCommand(String toAdd) {
         super(false);
+        this.toAdd = toAdd;
     }
 
     public String parseIngredientTitle(String input) {
         return input.replace("i/", "");
     }
 
     @Override
-    public void executeCommand(RecipeList recipes,IngredientList ingredients, String input) {
-        String ingredientTitle = parseIngredientTitle(input);
+    public void executeCommand(RecipeList recipes,IngredientList ingredients) {
+        String ingredientTitle = parseIngredientTitle(toAdd);
```

Comment:
since the above parseIngredientTitle() is removed, you pass the formatted parameters needed (in this case is ingredientTitle) directly in the parameter of this function